File: slader.py
import webbrowser
def slader():
    print "#####\tThis is slader script .\n It works until You write \"quit\" or something starts with q\t#####"
    page_number = "asd"
    while page_number.lower()[0] != "q" or page_number.lower() != "quit":
        page_number = raw_input("Page number please  \n ")
        question_number = raw_input("Question number please \n")
        slader = "http://www.slader.com/textbook/9780495011668-stewart-calculus-early-transcendentals-6th-edition/"+page_number+"/exercises/"+question_number+"/"
        webbrowser.open(slader)
slader()

File: slader.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int main(int argc, char const *argv[])
{
        printf("This is slader opener. Write \"exit\" to quit\n");
	char page[10],question[10];
        // Takes input of pages number
        printf("Page number: ");
	scanf("%s",page);
        // takes input of question number
	printf("Your page number is: %s\n",page );
        printf("Question number: ");
        scanf("%s",question);
	printf("Your question number: %s\n",question );
        // This is the basic link of slader. Concatenate our inputs to end of this string.
        char link[500]="http://www.slader.com/textbook/9780495011668-stewart-calculus-early-transcendentals-6th-edition/";
        // Concatenate them all
        strcat(link,page);
        strcat(link,"/exercises/");
        strcat(link,question);
        // Command is the xdg-open website-url to open website on terminal
        char command[] = "xdg-open ";
        strcat(command,link);
        system(command);
        


	return 0;
}
